python/pynessie/nessie_client.py

```diff
@@ -51,7 +51,12 @@ class NessieClient(object):
 
     def __init__(self: "NessieClient", config: confuse.Configuration) -> None:
         """Create a Nessie Client from known config."""
-        self._base_url = config["endpoint"].get()
+        url = config["endpoint"].get()
+        repo = config["repo"].get() if "repo" in config else None
+        if repo:
+            sep = "" if url[-1:] == "/" else "/"
```

Comment:
what does this mean?

Reply:
It's different whether your `endpoint` config value ends with a `/` or not.

Reply:
To produce a proper endpoint-URL and always separate the "raw base" URI and owner/repo part.

Reply:
got it, misread the first time